src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/CommonEnums.cs

```diff
@@ -0,0 +1,40 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models
+{
+    public enum ContainerType
+    {
+        AzureVM,
+    }
+
+    public enum BackupManagementType
+    {
+    }
+
+    public enum WorkloadType
+    {
+    }
+
+    public enum PsBackupProviderTypes
+    {
+        IaasVm = 1,
```

Comment:
Nit: Please use consistent casing for "VM" across. Also, if we are using an abbreviation and its length isn't >= 3, we have to use caps for the word. Please update PsBackupProviderTypes to PSBackupProviderTypes